README.md

```diff
@@ -1,67 +1,335 @@
 
-The fact is, code is for humans. So don't get too fancy with nesting as you will quickly find yourself in "nesting hell", where it is difficult for another programmer to read (and likely for you to read when you come back to it down the line.)
+```objc
+contacts[@"Uncle_Bob"][@"last_name"] = @"Gump";
+```
+
+This will cause an error that reads like this:
+
+```
+*** Terminating app due to uncaught exception  
+'NSInvalidArgumentException', reason: '-[__NSDictionaryI 
+setObject:forKeyedSubscript:]: unrecognized selector sent to 
+instance 0x7fd0e3eb6b60'
+```
+However, if we had initialized our sub-dictionaries as mutable dictionaries above, we'd be able to use the setter literal syntax. Let's change them with `mutableCopy`:
+
+```objc
+NSDictionary *contacts = @{ @"Jenny_Curran" : [@{ @"first_name"       : @"Jenny"               ,
+                                                  @"last_name"        : @"Curran"              ,
+                                                  @"relationship"     : @"Friend"              ,
+                                                  @"phone_number"     : @"(555) 867-5309"      ,
+                                                  @"email_address"    : @"jenny@email.com"     }
+                                               mutableCopy] ,
+                                
+                            @"Uncle_Bob"    : [@{ @"first_name"       : @"Bob"                 ,
+                                                  @"last_name"        : @""                    ,
+                                                  @"relationship"     : @"Uncle"               ,
+                                                  @"phone_number"     : @"(555) 876-1234"      ,
+                                                  @"email_address"    : @"unclebob@email.com"  }
+                                               mutableCopy] };
+```
+Now we can successfully correct Uncle Bob's last name:
+
+```objc
+contacts[@"Uncle_Bob"][@"last_name"] = @"Gump";
+    
+NSLog(@"%@", contacts[@"Uncle_Bob"][@"last_name"]);
+```
+This will print: `Gump`.
+
+### Nested Combinations
+
+Collections can be nested into collections of other types. While this isn't exclusive to arrays and dictionaries, the literal syntax available to these collections makes them the workhorses of managing large sets of data. Interacting with nested combinations is very common when working with internet API ("Application Program Interface") requests which you'll learn how to do later in the course.
+
+Take a look at this model schema of a response object from the [Cardstreams.io](https://developer.cardstreams.io/docs#!/Lifestreams/getComments) API. Despite being in JSON ("JavaScript Object Notation"), you should recognize the literals for strings (`""`), arrays (`[``]`), and dictionaries (`{``}`), all without the prepending `@` symbol native to Objective-C:
```

Comment:
Yeah I wouldn't bring JSON into this. Just talk about you have a soccer team with many players and each player is a dictionary and the players is an array